corehq/apps/cloudcare/views.py

```diff
@@ -471,20 +472,31 @@ def put(self, request, *args, **kwargs):
 def report_formplayer_error(request, domain):
     data = json.loads(request.body)
     error_type = data.get('type')
+    referer = request.META.get('HTTP_REFERER', '')
+
+    if '/cloudcare/apps/v2/' in referer:
```

Comment:
Hmm, I didn't know about that but it does seem reasonable. We unfortunately can't call it environment because we already use "`environment`" in datadog to mean `production|staging|...`

Reply:
Ah, I forgot about that, maybe `formplayer_environment`?

Reply:
I went with `cloudcare_env`, hope that's okay